cypress/e2e/pages/cytAssist.cy.ts

```diff
@@ -0,0 +1,217 @@
+import { SlotCopyMutation, SlotCopyMutationVariables } from '../../../src/types/sdk';
+import { LabwareTypeName } from '../../../src/types/stan';
+
+describe('CytAssist Page', () => {
+  before(() => {
+    cy.visit('/lab/cytassist');
+  });
+
+  describe('On load', () => {
+    it('shows a Visium LP CytAssist slide for the output', () => {
+      cy.findAllByText(/Visium LP CytAssist/i).should('have.length.above', 0);
+    });
+
+    it('disables the Save button', () => {
+      saveButton().should('be.disabled');
+    });
+  });
+
+  context('When user selects Visium LP CytAssist XL labwareType', () => {
+    before(() => {
+      selectLabwareType(LabwareTypeName.VISIUM_LP_CYTASSIST_XL);
+    });
+    it('shows a Visium LP CytAssist XL slide for the output', () => {
+      cy.findAllByText(/Visium LP CytAssist XL/i).should('have.length.above', 0);
+    });
+  });
+
+  context('When a user scans in a TP Slide', () => {
+    before(() => {
+      selectLabwareType(LabwareTypeName.VISIUM_LP_CYTASSIST);
+      cy.get('#labwareScanInput').type('STAN-3100{enter}');
+    });
+
+    it('shows it on the page', () => {
+      cy.findByText('STAN-3100').should('be.visible');
+    });
+
+    it('keeps the Save button disabled', () => {
+      saveButton().should('be.disabled');
+    });
+
+    context('When user selects some source slots', () => {
+      before(() => {
+        cy.get('#inputLabwares').within(() => {
+          cy.findByText('A1').click();
+          cy.findByText('B1').click({ shiftKey: true });
+        });
+      });
+
+      it('displays the table with A1 slot', () => {
+        cy.findByTestId('mapping_table').contains('td', 'A1');
+      });
+      it('displays the table with D1 slot', () => {
```

Comment:
Is this suppose to say 'B1' instead of 'D1'?

Reply:
Yes, I will change. thank you